src/components/ChatContent/ChatContent.js

```diff
@@ -364,20 +366,40 @@ export default class ChatContent extends React.Component {
     return this.state.isQueryRunning || this.state.isDrilldownRunning
   }
 
+  shouldHideQueryInputComponent = () => {
+    if (this.props.hideChatBarAfterInitialResponse && this.state.messages.length > 0) {
```

Comment:
const { hideChatBarAfterInitialResponse, shouldRender } = this.props
 const { messages } = this.state
 How about this?

Reply:
Updated